hasjob/views/index.py

```diff
@@ -33,13 +33,13 @@ def stickie_dict(post, url, pinned=False, show_viewcounts=False, show_pay=False,
         }
     if show_viewcounts:
         result['viewcounts'] = {
-            'listed': post.viewcounts['impressions'],
-            'viewed': post.viewcounts['viewed'],
-            'opened': post.viewcounts['opened'],
-            'applied': post.viewcounts['applied']
+            'listed': get_post_viewcounts(post)['impressions'],
+            'viewed': get_post_viewcounts(post)['viewed'],
+            'opened': get_post_viewcounts(post)['opened'],
+            'applied': get_post_viewcounts(post)['applied']
```

Comment:
You're no longer using `@cached_property` so this will hit cache multiple times. Call it once and extract values.